lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -744,6 +747,21 @@ class DropDown {
     }
   }
 
+  /**
+   * Focus out handler for the drop down menu.
+   * @param {Event} e
+   */
+  onMenuFocusOut(e) {
+    // The focusout event is not supported in our current version of typescript (3.5.3)
```

Comment:
```suggestion
    // TODO: The focusout event is not supported in our current version of typescript (3.5.3)
```

Let's just handle this later.

Reply:
Done!